server/routes/serviceProviderReferrals/addActionPlanActivitiesPresenter.test.ts

```diff
@@ -46,205 +46,159 @@ describe(AddActionPlanActivitiesPresenter, () => {
-          )
-
-          expect(presenter.text.title).toEqual('Add activity 1 to action plan')
-        })
-      })
-
-      describe('when 1 activity has been added', () => {
-        it('specifies the number of the activity to be added', () => {
-          const socialInclusionServiceCategory = serviceCategoryFactory.build({ name: 'social inclusion' })
-          const actionPlan = actionPlanFactory.justCreated(sentReferral.id).build({
-            activities: [
-              {
-                id: '91e7ceab-74fd-45d8-97c8-ec58844618dd',
-                description: 'Attend training course',
-                desiredOutcome: {
-                  id: '301ead30-30a4-4c7c-8296-2768abfb59b5',
-                  description:
-                    'All barriers, as identified in the Service User Action Plan (for example financial, behavioural, physical, mental or offence-type related), to obtaining or sustaining accommodation are successfully removed',
-                },
-                createdAt: '2020-12-07T20:45:21.986389Z',
+      it('specifies the number of the activity to be added', () => {
+        const socialInclusionServiceCategory = serviceCategoryFactory.build({ name: 'social inclusion' })
+        const actionPlan = actionPlanFactory.justCreated(sentReferral.id).build({
+          activities: [
+            {
+              id: '91e7ceab-74fd-45d8-97c8-ec58844618dd',
+              description: 'Attend training course',
+              desiredOutcome: {
+                id: '301ead30-30a4-4c7c-8296-2768abfb59b5',
+                description:
+                  'All barriers, as identified in the Service User Action Plan (for example financial, behavioural, physical, mental or offence-type related), to obtaining or sustaining accommodation are successfully removed',
               },
-            ],
-          })
-          const presenter = new AddActionPlanActivitiesPresenter(
-            sentReferral,
-            [socialInclusionServiceCategory],
-            actionPlan
-          )
-
-          expect(presenter.text.title).toEqual('Add activity 2 to action plan')
+              createdAt: '2020-12-07T20:45:21.986389Z',
+            },
+          ],
         })
+        const presenter = new AddActionPlanActivitiesPresenter(
+          sentReferral,
+          [socialInclusionServiceCategory],
+          actionPlan
+        )
+
+        expect(presenter.text.title).toEqual('Add activity 2 to action plan')
       })
     })
 
-    describe('subTitle', () => {
+    describe('referredOutcomesHeader', () => {
       it('includes the name of the service user', () => {
         const actionPlan = actionPlanFactory.justCreated(sentReferral.id).build()
 
         const presenter = new AddActionPlanActivitiesPresenter(sentReferral, serviceCategories, actionPlan)
 
-        expect(presenter.text.subTitle).toEqual('Referred outcomes for Jenny')
+        expect(presenter.text.referredOutcomesHeader).toEqual('Referred outcomes for Jenny')
```

Comment:
Similarly, could this be in the previous commit?

Reply:
I've lazily squashed those two commits together, it kind of makes sense to do them in one go.